### PR TITLE
#8 issue 'Error if custom field is missing' is resolved.

### PR DESCRIPTION
**Pull request by SmruthyJoseph**
#11 **Error if custom field is missing** issue is resolved.

### CODE REVIEW

File: wp-content/themes/soraytec/front-page.php [hunk 1]
```diff
@@ -4,19 +4,27 @@
...
+                <?php if($banner['description']): ?>
                     <p><?php echo $banner['description']; ?></p>
+                <?php endif; ?>
                 </div>
                 <div class="col-md-6 right">
+                <?php if($banner_img): ?>
                     <figure>
                         <img src="<?php echo $banner_img['url']; ?>" alt="">
                     </figure>
+                <?php endif; ?>
```
Comment (AbhijithHaridas) [inline wp-content/themes/soraytec/front-page.php:27]: Chance of error.
Minimize the number of **if**

File: wp-content/themes/soraytec/front-page.php [hunk 2]
```diff
@@ -107,10 +115,14 @@
...
     =================================================== -->
-    <?php $icontitle = get_field('logo_block'); ?>
+    <?php if(get_field('logo_block')): ?>
+        <?php $icontitle = get_field('logo_block'); ?>
+    <?php endif; ?>
     <section class="soraytec-ecosystem">
         <div class="container">
+        
             <h2><?php echo $icontitle['logo_title']; ?></h2>
+
```
Comment (AbhijithHaridas) [inline wp-content/themes/soraytec/front-page.php:125]: Chance of error.
Minimize the number of **if**

**Review by AbhijithHaridas — CHANGES_REQUESTED** [review]
@SmruthyJoseph Cleanup the **PR** changes so that it only has the changes to resolve this issue

File: wp-content/themes/soraytec/front-page.php (hunk 1 above)
Comment (SmruthyJoseph) [reply wp-content/themes/soraytec/front-page.php:27]: ok

File: wp-content/themes/soraytec/front-page.php (hunk 2 above)
Comment (SmruthyJoseph) [reply wp-content/themes/soraytec/front-page.php:125]: ok

**Comment by SmruthyJoseph** [comment]
#11 **Error if custom field is missing** issue corrected.

**Review by AbhijithHaridas — CHANGES_REQUESTED** [review]
Update the if scop and correct the if indentation

**Review by AbhijithHaridas — APPROVED** [review]

**Pull request merged**